server/DHT/simpleHash.go

```diff
@@ -0,0 +1,17 @@
+package DHT
+
+// Sum Ascii values in given string
+func sumChars(input string) int {
+	var sum = 0
+	for _, elem := range input {
+		sum += int(elem)
+	}
+	return sum
+}
+
+// Create simple hash of string by summing Ascii values then mod
+// by capacity
+func simpleASCIIHash(input string, capacity int) int {
```

Comment:
Done. However, crc32 returns uint32 and I'm currently casting to an int. A lesson on the finer points of using unsigned vs signed vs different sizes of ints as it relates to hashtables may be in order.